finagle-memcachedx/src/test/scala/com/twitter/finagle/memcachedx/unit/protocol/text/client/DecodingToResponseTest.scala

```diff
@@ -89,4 +89,15 @@ class DecodingToResponseTest extends FunSuite {
     assert(error.cause.getMessage() === errorMessage)
   }
 
+  test("parseResponse SERVER_ERROR") {
+    val context = new Context
+    import context._
+
+    val errorMessage = "sad panda error"
+    val buffer = Tokens( Seq("SERVER_ERROR", "sad", "panda", "error") map { Buf.Utf8(_) } )
+    val error = decodingToResponse.decode(null, null, buffer).asInstanceOf[protocol.Error]
+    assert(error.cause.getClass === classOf[ServerError])
```

Comment:
no need for triple equals, == is fine.  scalatest 2.x has an assert macro that's the :bomb: